modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractJavaCodegen.java

```diff
@@ -942,20 +942,88 @@ public String getAlias(String name) {
         return name;
     }
 
+    /**
+     * Return the default value of array property
+     * <p>
+     * Return null if there's no default value.
+     * Any non-null value will cause {{#defaultValue} check to pass.
+     *
+     * @param cp Codegen property
+     * @param schema Property schema
+     * @return string presentation of the default value of the property
+     */
+    public String toArrayDefaultValue(CodegenProperty cp, Schema schema) {
+        if (schema.getDefault() != null) { // has default value
+            if (cp.isArray && !cp.getUniqueItems()) { // array
+                List<String> _values = new ArrayList<>();
+
+                if (schema.getDefault() instanceof ArrayNode) { // array of default values
+                    ArrayNode _default = (ArrayNode) schema.getDefault();
+                    if (_default.isEmpty()) {
+                        return "null";
+                    }
+
+                    List<String> final_values = _values;
+                    _default.elements().forEachRemaining((element) -> {
+                        final_values.add(element.asText());
+                    });
+                } else { // single value
+                    _values = Arrays.asList(String.valueOf(schema.getDefault()));
```

Comment:
```suggestion
                    _values = java.util.Collections.singletonList(String.valueOf(schema.getDefault()));
```
OTOH you already created the `_values` list, so you could also just `add` here.